.github/CODEOWNERS

```diff
@@ -220,3 +230,12 @@
 /pkg/util/stop               @cockroachdb/server-prs
 /pkg/util/tracing            @cockroachdb/obs-inf-prs
 /pkg/workload/               @cockroachdb/sql-experience
+
+# Own all bazel files to dev-inf.
+**/BUILD.bazel               @cockroachdb/dev-inf
```

Comment:
This seems like it would be pretty noisy for dev-inf.  Every dependency change in a package can result in a BUILD.bazel update from `make bazel-generate`

Reply:
BUILD.bazel to me should be owned by the team that directory belongs to.
they should be modifying, e.g., the test dependencies in future, not dev-inf.

Reply:
It's a bit awkward - I don't think we want to request reviews based on changes to BUILD.bazel. Sure, teams kind of own them, but don't really know how to make manual changes to them. If they make manual changes, surely dev-inf should take a look. I agree that it is bad that all codegen-updates ping dev-inf, so best I can come up with is `dev-inf-noreview`. We could also make them unowned which has sort of the same effect in practice.

The reason I don't want to own them to the teams is that any time Ricky goes and changes something, the team doesn't care - dev-inf cares - and yet the team gets a ping. Maybe this is a small enough issue and we can drop this line, but there will be situations where all bazel files change at once.

Reply:
i think in that case we should do `-no-review`, at least for now.
i can buy the education argument :) one day i do imagine it belongs to the team